etc/kytos/logging.ini

```diff
@@ -1,38 +1,46 @@
 [formatters]
-keys: precise,brief
- 
+keys: precise,brief,thread
+
 [handlers]
 keys: console,syslog
- 
+
 [loggers]
-keys: root,daemon,cli
- 
+keys: root,daemon,cli,api_server
+
 [formatter_brief]
 format: %(name)s:%(levelname)s:  %(message)s
- 
+
 [formatter_precise]
 format: %(name)s:%(levelname)s %(module)s:%(lineno)d:  %(message)s
- 
+
+[formatter_thread]
+format: %(asctime)s - %(levelname)s [%(name)s] (%(threadName)s) %(message)s
+
 [handler_console]
 class: StreamHandler
-args: []
-formatter: brief
- 
+args:[sys.stdout]
+formatter: thread
+
 [handler_syslog]
 class: handlers.SysLogHandler
-args: [('localhost', handlers.SYSLOG_UDP_PORT), handlers.SysLogHandler.LOG_USER]
+args: ['/dev/log']
```

Comment:
I am using that http://scottbarnham.com/blog/2008/01/01/sysloghandler-not-writing-to-syslog-with-python-logging/, I can use syslog using logging.ini yet.